src/models/users.model.js

```diff
@@ -0,0 +1,124 @@
+const mongoose = require("mongoose");
+
+const userSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    minlenght: 3,
+    maxlenght: 25,
+    required: true,
+  },
+  lastname: {
+    type: String,
+    minlenght: 3,
+    maxlenght: 25,
+    required: true,
+  },
+  username: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  email: {
+    type: String,
+    required: true,
+  },
+  address: {
+    type: {
+      city: {
+        type: String,
+      },
+      suburb: {
+        type: String,
+      },
+      street: {
+        type: String,
+      },
+      number: {
+        type: Number,

```

Comment:
Yo vivo en el E1203 por ejemplo
```suggestion
        type: String,

```